.github/CONTRIBUTING.md

```diff
@@ -540,7 +540,113 @@ For more on good git commits, see (among _many_ other):
+
+> This PR closes [#1427](https://github.com/kirbydesign/designsystem/issues/1427), closes [#1428](https://github.com/kirbydesign/designsystem/issues/1428), closes [#1429](https://github.com/kirbydesign/designsystem/issues/1429)
+
+### The good pull-request has a logical commit history
+
+Maintaining a logical commit history constructed using [good commits](#commit) helps the reviewer easily get an overview and understanding of the changes contained in a pull-request.
+
+If you are having trouble maintaining a clean commit history, you could eventually try writing [pre-emptive commit comments](https://arialdomartini.wordpress.com/2012/09/03/pre-emptive-commit-comments/). This consists of writing down the commit comment, before writing your code. This can help you stay focused on a single task at a time. 
+
+![](images/good-commit-history.png)
+
+### The good pull-request only alters files related to the issue at hand
+
+While you might get the urge to be a good girl/boy scout and reorder imports or fix formatting in files not related to your issue, don't.
+
+While it is very much appreciated that you want to leave the code better than you found it, doing so in unrelated files will cause the git history of the project to be illogical. 
+For example, it would not make sense if the imports for the list component located in `list.component.ts` is reordered as part of a pull-request concerning optimizing a build step in the CI/CD pipeline. 
+
+Should you stumble upon possible optimizations or things that could be done better, go ahead and create new issues for these.
+
+### The good pull-request knows what it wants
+
+Tell the reviewer what you want to get out of your pull-request. 
+Should the reviewer do a full-on review or give initial guidance on a WIP implementation? Mention it in the pull-request so the reviewer knows what you are looking for and does not waste their time doing a full review on a WIP implementation. 
+
+As a convention in Kirby we signal which kind of feedback we want by creating a draft pull-request if it is a WIP implementation - and a normal one for a full review.
+
+Mention people you want to include in the discussion of your pull-request by using `@` , clarify why: 
+
+> @MadsBuchmann for details on the fontsource dependency
+
+### The good pull-requests only uses review comments for actual review comments
+
+Review comments are for review comments. Not for explaining your code. 
+
+If you feel the urge to add a review comment to clarify a hard to understand part of your code, it is not only the reviewer that will need that information. 
+Future contributors reading the code will also need it and could have a hard time understanding your code without. 
+
+Either refactor the code to make it more readable or add clarifying comments - it should be possible to understand the code without looking up the pull-request.
```

Comment:
Maybe emphasise that we mean "clarifying comments" _in the code itself_ - not to be confused with _review comments_

Reply:
Good catch, it can be a bit ambiguous.